tests_e2e/tests/ext_sequencing/ext_sequencing.py

```diff
@@ -159,7 +159,7 @@ def run(self):
         for instance in instances_ip_address:
             ssh_clients[instance.instance_name] = SshClient(ip_address=instance.ip_address, username=self._context.username, identity_file=self._context.identity_file)
 
-        if not VmExtensionIds.AzureMonitorLinuxAgent.supports_distro(next(iter(ssh_clients.values())).run_command("uname -a")):
```

Comment:
We use "uname -a" to check the distro in multiple places in the code. I am concerned with updating only this one instance.

Actually, maybe we should use get_distro(), in the Agent's code to do this

Reply:
Updated to use get_distro to get distro name and version for all test suites which use "uname -a"